seed/views/v3/tax_lot_properties.py

```diff
@@ -142,7 +142,9 @@ def export(self, request):
         data = TaxLotProperty.get_related(model_views, column_ids, columns_from_database)
 
         # add labels and notes
-        include_notes = request.data.get('include_notes')
+        include_notes = True
+        if 'include_notes' in request.data.keys():
+            include_notes = request.data.get('include_notes')
```

Comment:
This works as expected but `get` also has an optional second arg that specifies a default that you can throw `True` into and avoid the extra code.

Reply:
I do like that more.  Pushing a new branch soon